.github/CODEOWNERS

```diff
@@ -4,6 +4,10 @@
 # Migrations
 /src/sentry/migrations/  @getsentry/owners-migrations
 
+# API Owners for unowned endpoints
+endpoints/                      @getsentry/owners-api
```

Comment:
We could also add `src/sentry/api/base.py` here to be more thorough, if we wanted this group to be tagged on any unowned serializer changes, as well.

Reply:
done!